src/components/interface/MellowLpDepositForm/components/MellowLpDepositFormHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { Panel, Typography } from "src/components/atomic";
+import { SystemStyleObject, Theme, Box } from "@mui/system";
+import { ReactComponent as Mellow } from '../../MellowLPTable/components/mellow-icon.svg';
+import { Button } from "@mui/material";
+
+export type MellowLpDepositFormHeaderProps = {
+    onCancel: () => void;
+};
+
+const MellowLpDepositHeaderForm: React.FunctionComponent<MellowLpDepositFormHeaderProps> = ({ onCancel }) => {
+
+    const titleStyles: SystemStyleObject<Theme> = {
+        fontSize: '40px',
+        lineHeight: '1.2',
+        color: '#E5E1F9',
+        fontWeight: '700',
+        marginLeft: "16px",
+    };
+
+    const copyStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: '#9B97AD',
+        marginTop: "8px",
+    }
+
+    const tagStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: "#FF4AA9"
+    };
+
+    const boxStyles: SystemStyleObject<Theme> = {
+        display: "flex",
+        borderRadius: "4px",
+        padding: "4px 8px",
+        marginTop: "32px",
+        marginLeft: "0px",
+        width: "100px",
+        justifyContent: "center",
+    }
```

Comment:
move the style into ./styles.ts file in the same directory as the component. It will reduce the complexity and improve readability here.